airflow/executors/base_executor.py

```diff
@@ -14,16 +14,17 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
-"""Base executor."""
+"""
+Base executor - this is base class for all the implemented executors.
```

Comment:
typo? Base executor - this is **the** base class for all the implemented executors.

Reply:
Yep